demos/remote_attestation/init_ra_flow/dynamic_config.json

```diff
@@ -0,0 +1,15 @@
+{
+    "verify_mr_enclave" : "off",
```

Comment:
We could set those default values to "on" to help the user to do the right thing without mistake.